src/Dapr.Actors/Runtime/ConverterUtils.cs

```diff
@@ -11,6 +11,11 @@ internal class ConverterUtils
     {
         public static TimeSpan ConvertTimeSpanFromDaprFormat(string valueString)
         {
+            if (string.IsNullOrEmpty(valueString))
+            {
+                return default; // TODO: Should that be TimeSpan.FromMilliseconds(-1); DAPR spec says -1ms == never
```

Comment:
yes, it could be -1, it would happen for period, for dueTime , it would never be empty as SDK will pass a value and dapr will return it.